tensorflow/python/kernel_tests/sampling_ops_test.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.ops import sampling_ops
+from tensorflow.python.ops import gen_sampling_ops
```

Comment:
see comment below; here you should be able to call sampling_ops.bernoulli_sample, etc.